src/azure-cli/azure/cli/command_modules/ams/operations/sp.py

```diff
@@ -336,7 +336,7 @@ def _build_sp_result(subscription_id, location, resource_group_name, account_nam
     result = {
         'SubscriptionId': subscription_id,
         'ServicePrincipalName': sp_name,
-        'Region': location,
+        'Location': location,
```

Comment:
this will be a breaking change to end users

Reply:
Can you keep `Region` as well to avoid breaking changes.